discourseparsing/convert_rst_discourse_tb.py

```diff
@@ -1,222 +1,244 @@
-                              have a subdirectory parsed/mrg/wsj.')
+file_mapping = {"file1.edus": "wsj_0764.out.edus",
+                "file2.edus": "wsj_0430.out.edus",
+                "file3.edus": "wsj_0766.out.edus",
+                "file4.edus": "wsj_0778.out.edus",
+                "file5.edus": "wsj_2172.out.edus"}
+
+
+def main():  # noqa: D103
+    parser = argparse.ArgumentParser(description=__doc__,
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("rst_discourse_treebank_dir",
+                        help="Directory for the RST Discourse Treebank "
+                             "containing the subdirectory "
+                             "`data/RSTtrees-WSJ-main-1.0`.")
+    parser.add_argument("penn_treebank_dir",
+                        help="Directory for the Penn Treebank containing "
+                             "the subdirectory `parsed/mrg/wsj`.")
     parser.add_argument('--output_dir',
-                        help='directory where the output JSON files go.',
+                        help="Directory where the output JSON files will "
+                             "be saved.",
                         default='.')
     args = parser.parse_args()
 
     logging.basicConfig(format=('%(asctime)s - %(name)s - %(levelname)s - ' +
-                                '%(message)s'), level=logging.INFO)
-
-    logging.warning(
-        " Warnings related to minor issues that are difficult to resolve " +
-        " will be logged for the following files: " +
-        " file1.edus, file5.edus, wsj_0678.out.edus, and wsj_2343.out.edus." +
-        " Multiple warnings 'not enough syntax trees' will be produced" +
-        " because the RSTDTB has footers that are not in the PTB (e.g.," +
-        " indicating where a story is written). Also, there are some loose" +
-        " match warnings because of differences in formatting between" +
-        " treebanks.")
-
-    for dataset in ['TRAINING', 'TEST']:
+                                '%(message)s'),
+                        level=logging.INFO)
+
+    logging.warning("Warnings related to minor issues that are difficult to "
+                    "resolve will be logged for the following files: "
+                    "file1.edus, file5.edus, wsj_0678.out.edus, and "
+                    "wsj_2343.out.edus. Multiple warnings 'not enough syntax "
+                    "trees' will be produced because the RSTDTB has footers "
+                    "that are not in the PTB (e.g., indicating where a story "
+                    "is written). Also, there are some loose match warnings "
+                    "because of differences in formatting between the "
+                    "treebanks.")
+
+    # load the maxent part-of-speech tagger from NLTK which
+    # will be used to tag the RST treebank sentences
+    tagger = nltk.data.load("taggers/maxent_treebank_pos_tagger/PY3/english.pickle")
```

Comment:
Yes, because that was the default POS tagger with the NLTK version that Mike used and, actually, the results are slightly better with this tagger than with the averaged perceptron tagger which is the new default.